backend/BusinessLogic/Services/VideoService/VideoService.cs

```diff
@@ -191,6 +191,13 @@ await using (var stream = new FileStream(zipCreatePath, FileMode.Open))
             return memory;
         }
 
+        public async Task<long> GetUserVideosSize(User user)
```

Comment:
Should we pass only the userId through the parameters?

Reply:
it's also possible, could do that